mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -222,14 +178,18 @@ def _get_credential_infos(self, request_message_class, run_id, paths):
 
         return credential_infos
 
-    def _get_write_credential_infos(self, run_id, paths):
+    def _get_write_credential_infos(self, paths):
```

Comment:
Many artifact repositories don't have `run_id`s. Hence we're inlining it into the function rather than having it passed in.